go/test/endtoend/vtgate/foreignkey/main_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2023 The Vitess Authors.
+Copyright 2021 The Vitess Authors.
```

Comment:
wut?

Reply:
Sorry 😢 This was me! 
I copied a `main_test.go` from a different package and forgot to revert this change!

Reply:
Fixed.